documentation/assemblies/deploying/assembly-deploy-kafka-connect-with-plugins.adoc

```diff
@@ -5,7 +5,7 @@
 [id='using-kafka-connect-with-plug-ins-{context}']
 = Extending Kafka Connect with connector plugins
 
-The Strimzi container images for Kafka Connect include two built-in file connectors for moving file-based data into and out of your Kafka cluster.
+Up until Apache Kafka 3.1.0, the Strimzi container images for Kafka Connect include two built-in file connectors for moving file-based data into and out of your Kafka cluster.
```

Comment:
I'm not sure we need lines 8-25 now? 
If we want to keep it, do we need the versions?
Maybe this would be sufficient for the intro:

```
Kafka provides example file connectors for moving file-based data into and out of your Kafka cluster. 

.File connectors
[cols="2*",options="header",stripes="none",separator=¦]
|===
¦File Connector
¦Description
m¦FileStreamSourceConnector
¦Transfers data to your Kafka cluster from a file (the source).
m¦FileStreamSinkConnector
¦Transfers data from your Kafka cluster to a file (the sink).

|===

The procedures in this section describe how you can add the example connectors or your own connectors by doing one of the following:
```